[BackwardCompatibility] Remove aten::to from allow_list

hlu1 opened the pull request:
Summary: Remove aten::to from allow_list now that the aten::to schema change has landed (D29121620 (https://github.com/pytorch/pytorch/commit/eda2ddb5b06dce13bafd2a745e4634802e4640ef)).

Test Plan: CI

Differential Revision: D29187314

mruberry reviewed (changes requested):
Looks like this would still break the back compat test. I recommend just letting the  entry expire (which will happen naturally in a couple days)

Commit:
[BackwardCompatibility] Remove aten::to from allow_list (#60147)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/60147

Remove aten::to from allow_list now that the aten::to schema change has landed (D29121620 (https://github.com/pytorch/pytorch/commit/eda2ddb5b06dce13bafd2a745e4634802e4640ef)).

Test Plan: CI

Reviewed By: iseeyuan

Differential Revision: D29187314

fbshipit-source-id: 9b0a17f629fd03a146afe3140c7674d11e78d987
